.buildkite/auditbeat/auditbeat-pipeline.yml

```diff
@@ -62,7 +69,9 @@ steps:
 
   - label: ":linux: Load dynamic auditbeat pipeline"
     key: "auditbeat-pipeline"
-    command: ".buildkite/auditbeat/generate_auditbeat_pipeline.sh"
+    command: ".buildkite/scripts/generate_auditbeat_pipeline.sh"
```

Comment:
Should be: command: ".buildkite/auditbeat/generate_auditbeat_pipeline.sh"

Reply:
no, it should be the same as it is in the main